src/test/java/FooterTest.java

```diff
@@ -33,54 +31,61 @@ public void setUp() {
     }
 
     @Test
-    public void footerNavigation_FirstLevelCategories_CorrectlyDisplayed() {
+    public void footer_FooterLinks_CorrectlyDisplayed() {
         //then
         List<String> expectedNamesList = List.of("More websites for practice", "About us", "Customer Service", "Ask a question", "Write for Us",
                 "Search Terms", "Privacy and Cookie Policy", "Advanced Search", "Orders and Returns");
         List<String> expectedLinkList = List.of(TESTING_BOARD_URL + "q2a/4516/can-you-recommend-website-to-practice-selenium?show=4517#a4517",
                 HOMEPAGE_URL + "about-us", HOMEPAGE_URL + "customer-service", TESTING_BOARD_URL + "q2a/ask", TESTING_BOARD_URL + "write-for-us/",
                 HOMEPAGE_URL + "search/term/popular/", HOMEPAGE_URL + "privacy-policy-cookie-restriction-mode/", HOMEPAGE_URL + "catalogsearch/advanced/",
                 HOMEPAGE_URL + "sales/guest/form/");
-        validateFooterCategoriesNamesAndLinks(footerCategoriesList, expectedNamesList, expectedLinkList);
+        validateFooterCategoriesAndFooterNamesAndLinks(footerCategoriesList, expectedNamesList, expectedLinkList);
     }
 
     @Test
-    public void footer_SubscribeToNewsletterField_HasTextAndButtonCorrectlyDisplayed() {
+    public void footer_SubscribeToNewsletterForm_HasTextAndButtonCorrectlyDisplayed() {
         //then
         SoftAssertions softAssertions = new SoftAssertions();
         softAssertions.assertThat(footer.isSubscribeFormFieldDisplayed()).isTrue();
         softAssertions.assertThat(footer.isSubscribeButtonEnabled()).isTrue();
         softAssertions.assertThat(footer.getSubscribeButtonText()).isEqualTo("Subscribe");
         softAssertions.assertThat(footer.getEnterYourEmailPlaceHolderText()).isEqualTo("Enter your email address");
-        softAssertions.assertThat(footer.getSubscribeButtonFontColorAsHex()).isEqualTo(WHITE_COLOR);
-        softAssertions.assertThat(footer.getSubscribeButtonBackgroundColorAsHex()).isEqualTo(BLUE_COLOR);
+        softAssertions.assertThat(footer.getSubscribeButtonFontColorAsHex()).isEqualTo("#ffffff");
+        softAssertions.assertThat(footer.getSubscribeButtonBackgroundColorAsHex()).isEqualTo("#1979c3");
```

Comment:
Expected colors used here and in assertion above was used in other test classes as well.
It would be great to create a class that keeps all colors and use them in all test classes.